test/integration/apiserver/apply/apply_crd_test.go

```diff
@@ -720,3 +723,113 @@ spec:
 	}
 	verifyReplicas(t, result, 1)
 }
+
+// TestApplyCRDStructuralSchema tests that when a CRD has a structural schema in its validation field,
+// it will be used to construct the CR schema used by apply.
+func TestApplyCRDStatus(t *testing.T) {
+	defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.ServerSideApply, true)()
+
+	server, err := apiservertesting.StartTestServer(t, apiservertesting.NewDefaultTestServerOptions(), nil, framework.SharedEtcd())
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer server.TearDownFn()
+	config := server.ClientConfig
+
+	apiExtensionClient, err := clientset.NewForConfig(config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	dynamicClient, err := dynamic.NewForConfig(config)
+	if err != nil {
+		t.Fatal(err)
+	}
+	noxuDefinition := fixtures.NewMultipleVersionNoxuCRD(apiextensionsv1beta1.ClusterScoped)
+
+	var c apiextensionsv1beta1.CustomResourceValidation
+	err = json.Unmarshal([]byte(`{
+		"openAPIV3Schema": {
+			"type": "object",
+			"properties": {
+				"spec": {
+					"type": "object",
+					"properties": {
+						"ports": {
+							"type": "integer"
+						}
+					}
+				},
+				"status": {
+					"type": "object",
+					"properties": {
+						"conditions": {
+							"type": "array",
+							"x-kubernetes-list-type": "map",
+							"items": {
+								"type": "integer"
+							}
+						}
+					}
+				}
+			}
+		}
+	}`), &c)
+	if err != nil {
+		t.Fatal(err)
+	}
+	noxuDefinition.Spec.Validation = &c
+	falseBool := false
+	noxuDefinition.Spec.PreserveUnknownFields = &falseBool
+	noxuDefinition.Spec.Subresources = &apiextensionsv1beta1.CustomResourceSubresources{
+		Status: &apiextensionsv1beta1.CustomResourceSubresourceStatus{},
+	}
+
+	noxuDefinition, err = fixtures.CreateNewCustomResourceDefinition(noxuDefinition, apiExtensionClient, dynamicClient)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	kind := noxuDefinition.Spec.Names.Kind
+	apiVersion := noxuDefinition.Spec.Group + "/" + noxuDefinition.Spec.Version
```

Comment:
i'd use the GroupVersion constructor (not that the separator `/` will change in the future):
```
apiVersion := (schema.GroupVersion{
	Group: noxuDefinition.Spec.Group,
	Version: noxuDefinition.Spec.Version
}).String()	
```